terraform/routines.tf

```diff
@@ -1,4 +1,17 @@
+resource "google_bigquery_routine" "empty_to_null" {
+  dataset_id      = local.routines_dataset
+  definition_body = "NULLIF(NULLIF(NULLIF(TRIM(x), 'null'), 'None'), '')"
```

Comment:
Might be good to add a `LOWER` so casing doesn't matter

Reply:
yeah the issue though is that it will also affect the return value which may not want to be lowercased, could solve it with a case/when though 🤔